infra/helper.py

```diff
@@ -290,7 +292,13 @@ def _add_environment_args(parser):
                       help="set environment variable e.g. VAR=value")
 
 
-def _build_image(image_name, no_cache=False, pull=False):
+def _build_image_from_commit(image_name, commit, no_cache=False, pull=False):
+  """Builds a docker image from a specific commit SHA"""
+  _build_image('base-builder', commit=commit)
```

Comment:
why does base-builder need the commit value? shouldn't we pass it to the actual project image instead?

Reply:
The base builder needs the commit value specific to the repository it is going to build. That sets an environment variable that can be used by an image that inherits from base-builder.

Reply:
but it doesn't use that commit value as far as I understand. If that's correct, it should be enough to set up `COMMIT` env variable in the project image only, no?